services/fhir.ts

```diff
@@ -79,33 +80,91 @@ function getInactiveSearchParam(resourceType: string) {
     return {};
 }
 
+export async function createFHIRResource(
+    resource: AidboxResource,
+    searchParams?: SearchParams
+): Promise<RemoteDataResult<any>> {
+    return service(create(resource, searchParams));
+}
+
+export function create(resource: AidboxResource, searchParams?: SearchParams): AxiosRequestConfig {
+    return {
+        method: 'POST',
+        url: `/${resource.resourceType}`,
+        data: resource,
+        ...(searchParams ? { headers: { 'If-None-Exist': searchParams } } : {}),
+    };
+}
+
+export async function updateFHIRResource<R extends AidboxResource>(
+    resource: AidboxReference<R>,
+    searchParams?: SearchParams
+): Promise<RemoteDataResult<R>> {
+    return service(update(resource, searchParams));
+}
+
+export function update(resource: AidboxResource, searchParams?: SearchParams): AxiosRequestConfig {
+    const versionId = resource.meta && resource.meta.versionId;
+    return {
+        method: 'POST',
+        url: `/${resource.resourceType}/${resource.id}`,
```

Comment:
Url should be `/${resource.resourceType}/${resource.id}` if resource.id and searchParams passed. Otherwise, it should be `/${resource.resourceType}` according to fhir spec.

Reply:
And raise an exception if no resource.id and no searchParams passed.